File: C#/1557. Minimum Number of Vertices to Reach All Nodes.cs
public class Solution {
    public IList<int> FindSmallestSetOfVertices(int n, IList<IList<int>> edges) {
        List<int> ans=new List<int>();
        int[] check= new int[n];
        foreach(var e in edges){
            check[e[1]]++;
        }
        for (int i = 0; i < n; ++i){
            if (check[i] == 0){
                ans.Add(i);
            }
        }
        return ans;
    }
}

File: Python/1557. Minimum Number of Vertices to Reach All Nodes.py
class Solution:
    def findSmallestSetOfVertices(self, n: int, edges: List[List[int]]) -> List[int]:
        ans=[]
        check=[0]*n
        for e in edges:
            check[e[1]]=1
        
        for i in range(n):
            if check[i] == 0:
                ans.append(i)
            
        return ans
